services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeController.kt

```diff
@@ -182,7 +182,11 @@ When Hanke is created:
         if (hanke == null) {
             throw HankeArgumentException("No hanke given when creating hanke")
         }
-        val sanitizedHanke = hanke.copy(id = null, generated = false)
+        val sanitizedHanke =
+            hanke.apply {
+                id = null
+                generated = false
+            }
```

Comment:
sanitizedHanke is the same object as hanke, so the new value is redundant.
```suggestion
        hanke.id = null
        hanke.generated = false
```
Though I think this should be done in the service, not the controller. There's a logical place for it now that `createHankeInternal` was split from `createHanke`.

Reply:
Fixed as suggested. Actually the correct solution here will be (BE: Hankkeen luomiseen erillinen DTO) https://helsinkisolutionoffice.atlassian.net/browse/HAI-1893

So I will leave this "sanitation" in the controller for now.